cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/executor/ReplicationThrottleHelperTest.java

```diff
@@ -206,6 +215,81 @@ public void testAddingThrottlesWithPreExistingThrottles() throws InterruptedExce
     assertExpectedThrottledReplicas(kafkaZkClient, TOPIC1, "1:1");
   }
 
+  @Test
+  public void testDoNotModifyExistingWildcardReplicaThrottles() {
+    createTopics();
+
+    KafkaZkClient kafkaZkClient = KafkaCruiseControlUtils.createKafkaZkClient(zookeeper().connectionString(),
+        "ReplicationThrottleHelperTestMetricGroup", "AddingThrottlesWithNoPreExistingThrottles", false);
+
+    // Set replica throttle config values for both topics
+    setWildcardThrottleReplicaForTopic(kafkaZkClient, TOPIC0);
+    setWildcardThrottleReplicaForTopic(kafkaZkClient, TOPIC1);
+
+    final long throttleRate = 100L;
+
+    ReplicationThrottleHelper throttleHelper = new ReplicationThrottleHelper(kafkaZkClient, throttleRate);
+    ExecutionProposal proposal = new ExecutionProposal(
+        new TopicPartition(TOPIC0, 0),
+        100,
+        new ReplicaPlacementInfo(0),
+        Arrays.asList(new ReplicaPlacementInfo(0), new ReplicaPlacementInfo(1)),
+        Arrays.asList(new ReplicaPlacementInfo(0), new ReplicaPlacementInfo(2)));
+
+    ExecutionProposal proposal2 = new ExecutionProposal(
+        new TopicPartition(TOPIC0, 1),
+        100,
+        new ReplicaPlacementInfo(0),
+        Arrays.asList(new ReplicaPlacementInfo(0), new ReplicaPlacementInfo(3)),
+        Arrays.asList(new ReplicaPlacementInfo(0), new ReplicaPlacementInfo(2)));
```

Comment:
Nit: Bad indentation -- please align after `(` and refrain from putting the first argument to `ExecutionProposal` on a different line -- looks like the rest of this test (independent of this PR) also has issues with those.

Reply:
Fixed